hat/assets/js/apps/Iaso/domains/instances/compare/components/InstanceDetail.tsx

```diff
@@ -27,50 +15,59 @@ const InstanceDetail: FunctionComponent<Props> = ({ instanceId }) => {
-                    transparent
-                    padding={4}
-                    size={25}
-                />
-            </Box>
-        );
-    if (isError) {
-        return <ErrorPaperComponent message={formatMessage(MESSAGES.error)} />;
-    }
     return (
-        <>
-            <Box mb={4}>
-                <Typography variant="h5" color="secondary">
-                    {`${formatMessage(
-                        MESSAGES.submissionTitle,
-                    )} - ${instanceId}`}
-                </Typography>
-            </Box>
-            <WidgetPaper
-                expandable
-                isExpanded={false}
-                title={formatMessage(MESSAGES.infos)}
-                padded
-            >
-                <InstanceDetailsInfos currentInstance={data} />
-            </WidgetPaper>
-            <WidgetPaper
-                expandable
-                isExpanded={false}
-                title={formatMessage(MESSAGES.location)}
-            >
-                <InstanceDetailsLocation currentInstance={data} />
-            </WidgetPaper>
-            <WidgetPaper title={formatMessage(MESSAGES.form)}>
-                <InstanceFileContent instance={data} />
-            </WidgetPaper>
-        </>
+        <InstanceDetailRaw
+            data={data}
+            isLoading={isLoading}
+            isError={isError}
+        />
     );
+
+    // const { formatMessage } = useSafeIntl();
+
+    // if (isLoading)
+    //     return (
+    //         <Box height="70vh">
+    //             <LoadingSpinner
+    //                 fixed={false}
+    //                 transparent
+    //                 padding={4}
+    //                 size={25}
+    //             />
+    //         </Box>
+    //     );
+    // if (isError) {
+    //     return <ErrorPaperComponent message={formatMessage(MESSAGES.error)} />;
+    // }
+    // return (
+    //     <>
+    //         <Box mb={4}>
+    //             <Typography variant="h5" color="secondary">
+    //                 {`${formatMessage(
+    //                     MESSAGES.submissionTitle,
+    //                 )} - ${instanceId}`}
+    //             </Typography>
+    //         </Box>
+    //         <WidgetPaper
+    //             expandable
+    //             isExpanded={false}
+    //             title={formatMessage(MESSAGES.infos)}
+    //             padded
+    //         >
+    //             <InstanceDetailsInfos currentInstance={data} />
+    //         </WidgetPaper>
+    //         <WidgetPaper
+    //             expandable
+    //             isExpanded={false}
+    //             title={formatMessage(MESSAGES.location)}
+    //         >
+    //             <InstanceDetailsLocation currentInstance={data} />
+    //         </WidgetPaper>
+    //         <WidgetPaper title={formatMessage(MESSAGES.form)}>
+    //             <InstanceFileContent instance={data} />
+    //         </WidgetPaper>
+    //     </>
+    // );
```

Comment:
?

Reply:
If I had to pay you a beer every time I forget comments in my code, you'd be alcoholic